_episodes/01-short-introduction-to-Python.md

```diff
@@ -11,7 +11,10 @@ objectives:
     - "Perform mathematical operations in Python using basic operators."
     - "Define the following as it relates to Python: lists, tuples, and dictionaries."
 keypoints:
-    - "FIXME"
+    - "Python is an interpreted language which can be used interactively (executing one command at a time) or in scripting mode (executing a series of commands saved in file)."
+    - "One can assign a value to a variable in python. Those variables can be of several types (e.g. `str` for strings, `int` for integers, `float`s, `list`s, `tuple`s, and `dict` for dictionaries)."
```

Comment:
1. it's Python, not python (just like it's C and C++, not c and c++)
2. What is the purpose of the first sentence? Is it that Python has variables? If so, I'd vote for removing this first sentence because this is ... somewhat expected.
3. I'd suggest saying that `* Python supports several basic data types, such as integer, floating point and complex numbers, strings, and other. * Python provides other data types for storing multiple values together (lists and tuples) or for providing mappings between keys and values (dictionaries).`

I don't think abbreviations 'str', 'int', and so forth deserve their place in the keypoint

Reply:
The point of the first sentence was to say that Python is interpreted (as opposed to compiled).
I made changes to reflect your suggestions.